### ISSUE TITLE
Make golang work out of the box

### ISSUE THREAD

**Issue by csoriano1618** [flathub/com.visualstudio.code.oss#37]
Since this repository already has support for golang with the extension, it would be nice if it could work out of the box. For that, from [this issue](https://github.com/flathub/com.visualstudio.code/issues/40) seems that what is missing is just to put some env variables as part of the recipe:
```
--env=PATH=/app/bin:/usr/bin:/usr/lib/sdk/golang/bin --env=GOROOT=/usr/lib/sdk/golang --env=GOPATH=$HOME/go com.visualstudio.code
```

I can create a PR once master is fixed.

**Comment by amtlib-dot-dll** [comment]
Personally speaking most of the users do not use `golang`, so even those tools required by the official VSCode Golang extension are bundled separately from the default installation.

**Comment by csoriano1618** [comment]
That's true, however seems the containers world (that is becoming quite ubiquitous) is basically all Go, would be nice to have this ready. In any case, doing these modifications doesn't hurt other workflows, so probably is an easy win right?

**Comment by snaggen** [comment]
Setting these paths will get much of the go experience working. However I still can't use the debugger, I just get
 `could not launch process: fork/exec /home/snaggen/src/keytrainer/debug: operation not permitted`

**Comment by amtlib-dot-dll** [comment]
@snaggen Try --allow=devel?

**Comment by snaggen** [comment]
--allow=devel fixed it

**Comment by snaggen** [comment]
It seems that the use of $HOME for GOPATH depends on https://github.com/flatpak/flatpak/pull/2555 to work, the alternative is a wrapper script I guess.

**Comment by W73** [comment]
You may 
 `flatpak override --env=PATH=/app/bin:/usr/bin:/home/USERNAME/go/bin com.visualstudio.code.oss `

it create 
`ls -l /var/lib/flatpak/overrides/`

```
[Environment]
PATH=/app/bin:/usr/bin:/home/USERNAME/go/bin
```
And it find your go
